locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,561 @@
+
+# Page title
+#: home/service/search.py:170 templates/base/navigation.html:65
+#: templates/details_base.html:14
+msgid "Search"
+msgstr "Search"
+
+# Page title
+#: home/views.py:25 templates/base/navigation.html:60
+msgid "Home"
+msgstr "Home"
+
+# 404 page hint
+#: templates/404.html:13 templates/base/error/page_not_found.html:7
+msgid "If you typed the web address, check it is correct."
+msgstr "If you typed the web address, check it is correct."
+
+# 404 page hint
+#: templates/404.html:16 templates/base/error/page_not_found.html:10
+msgid "If you pasted the web address, check you copied the entire address."
+msgstr "If you pasted the web address, check you copied the entire address."
+
+# Error page message
+#: templates/500.html:13
+msgid "There is a problem with the server."
+msgstr "There is a problem with the server."
+
+# Error page message
+#: templates/500_datahub_unavailable.html:13
+msgid ""
+"The DataHub catalogue is currently unavailable. Please try again in a few "
+"minutes."
+msgstr ""
+"The DataHub catalogue is currently unavailable. Please try again in a few "
+"minutes."
+
+# Skip link
+#: templates/base/base.html:13
+msgid "Skip to main content"
+msgstr "Skip to main content"
+
+# Page heading
+#: templates/base/cookie.html:3 templates/base/cookie.html:7
+msgid "Cookies on Find MOJ data"
+msgstr "Cookies on Find MOJ data"
+
+# Cookie usage
+#: templates/base/cookie.html:9
+msgid ""
+"We use Google Analytics cookies so we can understand how you use the service "
+"and make improvements."
+msgstr ""
+"We use Google Analytics cookies so we can understand how you use the service "
+"and make improvements."
+
+# Footer link
+#: templates/base/footer.html:6
+msgid "Support links"
+msgstr "Support links"
+
+# Footer link
+#: templates/base/footer.html:10
+msgid "Help"
+msgstr "Help"
+
+# Footer link
+#: templates/base/footer.html:15
+msgid "Cookies"
+msgstr "Cookies"
+
+# Footer link
+#: templates/base/footer.html:20
+msgid "Contact"
+msgstr "Contact"
+
+# Footer link
+#: templates/base/footer.html:25
+msgid "Terms and conditions"
+msgstr "Terms and conditions"
+
+# OGL licence
+#: templates/base/footer.html:42
+#, python-format
+msgid ""
+"All content is available under the <a class=\"govuk-footer__link\" "
+"href=\"%(href)s\" rel=\"license\">Open Government Licence v3.0</a>, except "
+"where otherwise stated"
+msgstr ""
+"All content is available under the <a class=\"govuk-footer__link\" "
+"href=\"%(href)s\" rel=\"license\">Open Government Licence v3.0</a>, except "
+"where otherwise stated"
+
+# OGL licence link
+#: templates/base/footer.html:52
+#, python-format
+msgid ""
+"<a class=\"govuk-footer__link govuk-footer__copyright-logo\" "
+"href=\"%(href)s\">© Crown copyright</a>"
+msgstr ""
+"<a class=\"govuk-footer__link govuk-footer__copyright-logo\" "
+"href=\"%(href)s\">© Crown copyright</a>"
+
+# Service name - note the capitalisation
+#: templates/base/navigation.html:16 templates/home.html:11
+msgid "Find MOJ data"
```

Comment:
```suggestion
msgid "Find MoJ data"
```

Reply:
"MOJ" is the capitalisation we're using at the moment, with lowercase "data", although I'm not sure the reasoning. @seanprivett can you confirm this is correct?

Reply:
I'll sort out the IDs separately, maybe shorten everything to a programmatic key to avoid confusion